### ISSUE TITLE
NestedBlockDepth false negative

### ISSUE THREAD

**Issue by BraisGabin** [arturbosch/detekt#2085]
NestedBlockDepth should find an issue here:
```kotlin
    fun procedure(n: Int) {
        if (true) {
            if (true) {
            } else if (true) {
                if (true) {
                    if (true) {
                        if (true) { 
                            if (true) { 
                                if (true) {
                                }
                            }
                        }
                    }
                }
            }
        }
    }
```

But it doesn't. It seems that the `if else` is breaking something.

**Comment by schalkms** [comment]
Thanks for quickly fixing this!!
